src/beamcontact/4C_beamcontact_beam3contact_manager.cpp

```diff
@@ -925,16 +925,16 @@ void CONTACT::Beam3cmanager::init_beam_contact_discret()
       esdata, erdata, (int)ertproc.size(), ertproc.data(), bt_sol_discret().get_comm());
 
   // build completely overlapping node map (on participating processors)
-  std::shared_ptr<Epetra_Map> newnodecolmap = std::make_shared<Epetra_Map>(
-      -1, (int)rdata.size(), rdata.data(), 0, bt_sol_discret().get_comm());
+  std::shared_ptr<Epetra_Map> newnodecolmap = std::make_shared<Epetra_Map>(-1, (int)rdata.size(),
+      rdata.data(), 0, Core::Communication::as_epetra_comm(bt_sol_discret().get_comm()));
   sdata.clear();
   stproc.clear();
   rdata.clear();
   allproc.clear();
 
   // build completely overlapping element map (on participating processors)
-  std::shared_ptr<Epetra_Map> newelecolmap = std::make_shared<Epetra_Map>(
-      -1, (int)erdata.size(), erdata.data(), 0, bt_sol_discret().get_comm());
+  std::shared_ptr<Epetra_Map> newelecolmap = std::make_shared<Epetra_Map>(-1, (int)erdata.size(),
+      erdata.data(), 0, Core::Communication::as_epetra_comm(bt_sol_discret().get_comm()));
```

Comment:
Note that all conversions to `Epetra_Comm` when creating an Epetra_Map will vanish once we wrap this class.